scripts/build-images

```diff
@@ -30,7 +30,7 @@ xargs -n1 -t docker image pull --quiet << EOF >> build/images-core.txt
 EOF
 
 xargs -n1 -t docker image pull --quiet << EOF > build/images-canal.txt
-    ${REGISTRY}/rancher/hardened-calico:v3.28.0-build20240528
+    ${REGISTRY}/rancher/hardened-calico:v3.28.0-build20240625
     ${REGISTRY}/rancher/hardened-flannel:v0.25.3-build20240605
```

Comment:
The flannel version here needs to be bumped as well - `v3.28.0-build2024062502`  included a flannel bump but wasn't ever pulled through.
```
[[36mBASICS[m] -docker.io/rancher/hardened-flannel:v0.25.3-build20240605
[[36mBASICS[m] +docker.io/rancher/hardened-flannel:v0.25.4-build20240610
```

```suggestion
    ${REGISTRY}/rancher/hardened-flannel:v0.25.4-build20240610
```